src/pages/home/report/ReportActionCompose/ReportActionCompose.tsx

```diff
@@ -424,9 +434,9 @@ function ReportActionCompose({
                                         isScrollLikelyLayoutTriggered={isScrollLikelyLayoutTriggered}
                                         raiseIsScrollLikelyLayoutTriggered={raiseIsScrollLikelyLayoutTriggered}
                                         reportID={reportID}
-                                        parentReportID={report.parentReportID}
-                                        parentReportActionID={report.parentReportActionID}
-                                        includesChronos={ReportUtils.chatIncludesChronos(report)}
+                                        parentReportID={report?.parentReportID}
+                                        parentReportActionID={report?.parentReportActionID}
+                                        includeChronos={ReportUtils.chatIncludesChronos(report)}
```

Comment:
Please test this change

Reply:
https://github.com/Expensify/App/pull/34646/files#r1489665371 here is my response 😄